session/driver/file.go

```diff
@@ -61,9 +61,10 @@ func (f *File) Open(string, string) error {
 	return nil
 }
 
+//
```

Comment:
Good catch, thanks, could you remove this line?

Reply:
ye, i remove this line